pkg/utils/has/controller.go

```diff
@@ -268,6 +274,17 @@ func (h *SuiteController) ComponentDeleted(component *appservice.Component) wait
 	}
 }
 
+func (h *SuiteController) CheckForImageAnnotation(component *appservice.Component) wait.ConditionFunc {
+	return func() (bool, error) {
+		componentCR, err := h.GetHasComponent(component.Name, component.Namespace)
+		if err != nil {
+			return false, nil
```

Comment:
return error instead nil? in this case something is wrong when error is returned for GetHasComponent

Reply:
Agree. Done.